App/Sagas/TextileSagas.ts

```diff
@@ -149,7 +149,7 @@ function * processAvatarImage(uri: string, defaultThread: TT.Thread) {
 }
 
 export function * viewThread ( action: ActionType<typeof UIActions.viewThreadRequest> ) {
-  yield call(TextileNodeActions.refreshMessagesRequest, false)
+  yield put(TextileNodeActions.refreshMessagesRequest())
```

Comment:
Think this was a bug @andrewxhill , so I fixed it. The old version would do nothing.